src/Menus/ElementalGrid/ElementalGridMenu.php

```diff
@@ -49,26 +62,41 @@ public function isMenuSupported(): bool
         return (InstalledVersions::isInstalled('dnadesign/silverstripe-elemental') && $page->hasExtension(self::ELEMENTAL_GRID_PAGE_EXTENSION_CLASS));
     }
 
+    public function getElementalConfig(): ElementalConfig
+    {
+        return new ElementalConfig();
+    }
+
     public function getItems(): ArrayList
     {
         $page = Controller::curr()?->data();
-        $menuItems = [
-            CMSMenuItem::create()->setCustomHTML("<b>Elemental Grid</b>"),
-        ];
+
+        $menuItems = [];
 
         $elements = $page?->ElementalArea?->Elements();
```

Comment:
As you are looping over `$elements` make sure it is an array, now it can be null.

```suggestion
        $elements = $page?->ElementalArea?->Elements() ?? [];
```